sql3/planner/opquery.go

```diff
@@ -130,10 +131,13 @@ func (i *queryIterator) Next(ctx context.Context) (types.Row, error) {
 		// either error or no more rows, either way update the request
 		requestId, ok := fbcontext.RequestID(ctx)
 		if !ok {
-			return nil, sql3.NewErrInternalf("unable to get request id from context")
+			return nil, errors.Wrapf(sql3.NewErrInternalf("unable to get request id from context"), "next on child: %s", err)
```

Comment:
I'm not understanding this but I'm a bit confused by it, because it looks like we're creating a NewErrInternal, but we're also wrapping an existing err, and I'm just a bit unclear on the intended output message.

Reply:
... Also I'm not sure whether "err" is valid here, because it doesn't look like `err` was returned by the thing that we got the !ok from, so this seems like it could be an unrelated error?

Reply:
I was a little confused by this logic as well, but what I noticed while fixing the linter complaint was that if we were unable to get the request id from the context, then we were losing the original error... so this was my attempt to include both errors in the return.